bs-react-native-next/src/apis/Platform.rei

```diff
@@ -0,0 +1,9 @@
+type os = pri string;
```

Comment:
what does `pri string` means?

Reply:
`private` in OCaml (as suggested by Hongbo in https://github.com/BuckleScript/bucklescript/issues/3472#issuecomment-480190956) becomes `pri` in Reason.

Just `type os;` alone won't work as the inline constants are of type `string`.

Reply:
Thanks for the explanation.